plugins/europeana/entity.js

```diff
@@ -161,7 +161,7 @@ function getDataForEntities(entities, entityKey) {
   })
     .then((response) => {
       let items = response.data.items ? response.data.items : [];
-      return getRelatedEntityTitleLink(items.slice(0,10));
+      return getRelatedEntityTitleLink(items.slice(0,4));
```

Comment:
Could we slice earlier to prevent retrieving more data from the entity search than we need, i.e.
```js
const entityLabels = entities.slice(0,4).map(entity => {
```
(Note also that `entityLabels` should use `const` instead of `let`.)